node/src/command.rs

```diff
@@ -235,20 +236,42 @@ pub fn run() -> sc_cli::Result<()> {
 		}
 		Some(Subcommand::Revert(cmd)) => {
 			let runner = cli.create_runner(cmd)?;
-			runner.async_run(|mut config| {
-				let (client, backend, _import_queue, task_manager) = service::new_partial(&mut config)?;
-				Ok((cmd.run(client, backend), task_manager))
+			runner.async_run(|config| {
+				let (client, backend, _import_queue, task_manager) = new_partial(&config, true)?;
+				Ok((cmd.run(client, backend, None), task_manager))
 			})
 		}
 		Some(Subcommand::Benchmark(cmd)) => {
-			if cfg!(feature = "runtime-benchmarks") {
-				let runner = cli.create_runner(cmd)?;
+			let runner = cli.create_runner(cmd)?;
 
-				runner.sync_run(|config| cmd.run::<Block, service::BasiliskExecutorDispatch>(config))
-			} else {
-				Err("Benchmarking wasn't enabled when building the node. \
-				You can enable it with `--features runtime-benchmarks`."
-					.into())
+			match cmd {
+				BenchmarkCmd::Pallet(cmd) => {
```

Comment:
Does this mean that also the command to run benchmarks changed?

Reply:
otherwise it fails on this https://github.com/paritytech/polkadot/blob/bb1bf69dfc538abd40c8d614e8bbf46580d7005a/runtime/parachains/src/configuration.rs#L393